rust/src/sip/sip.rs

```diff
@@ -147,20 +148,75 @@ impl SIPState {
         }
     }
 
+    fn parse_request_tcp(
+        &mut self, flow: *const core::Flow, stream_slice: StreamSlice,
+    ) -> AppLayerResult {
+        let input = stream_slice.as_slice();
+        if input.is_empty() {
+            return AppLayerResult::ok();
+        }
+        let _pdu = Frame::new(
+            flow,
+            &stream_slice,
+            input,
+            input.len() as i64,
+            SIPFrameType::Pdu as u8,
```

Comment:
This frame does not look right :
A TCP sent here may have 0 or multiple SIP PDUs

This should be done after `sip_parse_request` returned `Ok`

Cf https://github.com/OISF/suricata/pull/10048 (MQTT current implementation is wrong)